internal/core/alias.go

```diff
@@ -0,0 +1,12 @@
+package core
+
+// aliasDisabled returns if alias are disabled for a command
+func aliasDisabled(cmd string) bool {
+	switch cmd {
+	case "alias":
+		return true
+	case "autocomplete":
+		return true
```

Comment:
nit

```suggestion
	case "alias","autocomplete":
		return true
```

I would also have added `"init"` in this list